server/channels/store/localcachelayer/layer.go

```diff
@@ -476,9 +482,25 @@ func (s *LocalCacheStore) doStandardReadCache(cache cache.Cache, key string, val
 	return err
 }
 
+func (s *LocalCacheStore) doMultiReadCache(cache cache.Cache, keys []string, values []any) []error {
+	errs := cache.GetMulti(keys, values)
```

Comment:
We could, but that would make the `GetMulti` method slightly different than `Get`, and moreover, it would push the hit/miss logic to the lower layer. This keeps the API simple and consistent, at the cost of an additional loop. I am considering to keep this as-is. LMK what you think.

Reply:
I see your point and I'd be in favor of consistency, but isn't this a hot path that whenever we hit to cache, we have to do an additional loop multiplied by size of the cache. Am I over estimating it's impact?

Reply:
It's all in-memory operations. There's no network roundtrip. I believe there's little to gain here.

Reply:
Gotcha, if you see the gains negligible it's fine then.